website/content/docs/internals/limits.mdx

```diff
@@ -63,6 +63,8 @@ The number of mount points can be monitored by reading the
 similar sub-paths for namespaces respectively, like: `namespace1/sys/auth`,
 `namespace1/sys/mounts`, etc.
 
+Alternatively, the [Vault telemetry metrics](/vault/docs/internals/telemetry) `vault.core.mount_table.num_entries` and `vault.core.mount_table.size` are available to monitor the number of mount points and size of each mount table, respectively. 
```

Comment:
Suggest linking directly to the metric definitions rather than the top-level metric page.

```suggestion
Alternatively, use the
[`vault.core.mount_table.num_entries`](/vault/docs/internals/telemetry/metrics/core-system#vault-core-mount_table-num_entries)
and
[`vault.core.mount_table.size`](/vault/docs/internals/telemetry/metrics/core-system#vault-core-mount_table-size)
telemetry metrics to monitor the number of mount points and size of each mount table.
```
style correction: active voice